deploy-package/action.yml

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+name: 'package-deploy'
+description: 'Power Platform package deploy - Deploys the package dll or the zip file with a package.'
```

Comment:
add verbiage that deploy is only supported on Windows runners; the action itself should fail but nice to tell this important detail upfront :-)